KJ1124/Thor_Puzzle_assign.c

```diff
@@ -0,0 +1,122 @@
+
+*/
+/* 
+Thor's Movement
+----------------
+**Based on the position light of power I first determined which direction he would travel and then developed this code on the basis of each test code.**
+
+**Test case 1 & 2:**
+---------------------
+
+This line of code is where you will put in how Thor will move. In order to observe what direction he will go in based on what you are already given click on the "play testcase" button in the bottom right hand corner and watch the video as he moves. After this you will change the direction that thor moves "E" (East) towards the light of power and you passed the first test case. You will notice that as you play each testcase the orientation of thor changes, so you can repeat the same process in testcase 01 (straight line) in test case 02 (Up) but this time go N North in the direction of the light of power.
+*/
+/* **Test case 3:**
++++++++++++++++++++
+
+Easy angle is where things get a little tricky. In order to make Thor move toward the light of power, while also being mindful of his "Energy" located under the console output, you must how thor move SW (South West) and W (West) at the same time so you will need 2 else if statements. The reason you must increment is to prevent Thor from moving out of the boundary conditions before he reaches the light of power.
+
+*/
+/* **Test case 4:**
+
+The final test case (Optimal angle) utilizing everything implemented before while also assuring that the whole code is properly looped so that you can play all testcases. Here is where you apply the use of if and else if statements to get the last angle to be short but also keep you from running out of energy. In this case in order to achieve the angle you desire you must increment thor's inital X and Y position in the SE (South East) direction.
+
+/* 
+*Note: In order to loop each of the test cases to work all at the same time you have to use an if statement so that you can input more than one direction for each time that Thor moves. You will notice that as you play each testcase. Your conditions are created by making the light of power position X or Y (light_x/light_y) greater than or less than the positon of Thor(initial_tx/initial_ty) in the X or Y.*
+
+Code for Movement
+------------------
+*/
+// **this is the fourth test case...**
+if((light_y > initial_ty) && (light_x > initial_tx))
+{
+    cout << "SE" << endl;
+    initial_tx++;
+    initial_ty++;
+}
+// **This is the first test case below...**
+else if(light_x > initial_tx) {
+    cout << "E" << endl;
+}
+// **This is the second test case...**
+else if(light_y < initial_ty) {
+    cout << "N" << endl;
+}
+// **This is the third test case...**
+else if(light_y > initial_ty) {
+    cout << "SW" << endl;
+    initial_ty++;
+}
+else if(light_x < initial_tx) {
+    cout << "W" << endl;
+    initial_tx--;
+} 
+// **Don't forget to include the "else{}" statment at the end.**
+else{};
+}
```

Comment:
Fix spacing -- all the code should be indented, so that each bracket is has its own horizontal position.